packages/react-mutation-mapper/src/util/MyCancerGenomeUtils.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { parseMyCancerGenomeLink } from './MyCancerGenomeUtils';
+import { assert } from 'chai';
+
+describe('MyCancerGenomeUtils', () => {
+    describe('parseMyCancerGenomeLink', () => {
+        it('extracts URL and text from a pre-formatted my cancer genome HTML string', () => {
+            const htmlLink =
+                '<a href="http://mycancergenome.org/content/disease/colorectal-cancer/kras/38/">KRAS c.37G>T (G13C) Mutation in Colorectal Cancer</a>';
+
+            const parsed = parseMyCancerGenomeLink(htmlLink);
+
+            assert.isNotNull(parsed);
+            assert.equal(
+                parsed!.text,
+                'KRAS c.37G>T (G13C) Mutation in Colorectal Cancer'
```

Comment:
Would it be better just put this as a constant?

Reply:
I just left the existing test as is, I think this is not too bad to leave like this.